features/request/RequestContext.php

```diff
@@ -474,12 +474,22 @@ public function iHaveProvidedAnRporiginOf($rpOrigin)
      */
     public function iCallMfaCreate()
     {
-        $this->getIdBrokerClient()->mfaCreate(
-            $this->requestData['employee_id'],
-            $this->requestData['type'],
-            $this->requestData['label'],
-            $this->rpOrigin
-        );
+        if (empty($this->requestData['recovery_email'])) {
+            $this->getIdBrokerClient()->mfaCreate(
+                $this->requestData['employee_id'],
+                $this->requestData['type'],
+                $this->requestData['label'],
+                $this->rpOrigin,
+            );
+        } else {
+            $this->getIdBrokerClient()->mfaCreate(
+                $this->requestData['employee_id'],
+                $this->requestData['type'],
+                $this->requestData['label'],
+                $this->rpOrigin,
+                $this->requestData['recovery_email'],
+            );
+        }
```

Comment:
Why not just do this?

```suggestion
        $this->getIdBrokerClient()->mfaCreate(
            $this->requestData['employee_id'],
            $this->requestData['type'],
            $this->requestData['label'],
            $this->rpOrigin,
            $this->requestData['recovery_email'],
        );
```

Reply:
The tests were failing since it was adding a `recovery_email: null` into the body.

Reply:
OK, that makes sense